lib/infrastructure/external_api_broker.rb

```diff
@@ -0,0 +1,63 @@
+require "rest_client"
+
+class ExternalApiBroker
```

Comment:
Maybe `ServiceCatalogClient` ?  I don't care too much about the name though since its going to change soon anyway